automatic/lightalloy/lightalloy.nuspec

```diff
@@ -4,24 +4,41 @@
   <metadata>
     <id>lightalloy</id>
     <title>Light Alloy Video Player</title>
-    <version>4.10.1</version>
+    <version>4.10.2</version>
```

Comment:
Wouldn't this prevent the update? I noticed the latest version available on chocolatey.org is `4.10.1`.